supplier_app/forms.py

```diff
@@ -93,4 +148,19 @@ class Meta:
             'bank_name': forms.TextInput(attrs={'class': 'form-control'}),
             'bank_code': forms.TextInput(attrs={'class': 'form-control'}),
             'account_number': forms.TextInput(attrs={'class': 'form-control'}),
+            'cbu_file': forms.FileInput(attrs={'accept': 'application/pdf'}),
         }
+
+    def is_valid(self):
+        valid = super().is_valid()
+        if not valid:
+            return valid
+
+        file_is_valid, msg = validate_file(
+            self.cleaned_data['cbu_file'],
+            MAX_SIZE_FILE,
+            ALLOWED_FILE_EXTENSIONS
+        )
```

Comment:
Do we need a CBU file? If that's the case, should constants `MAX_SIZE_FILE` and `ALLOWED_FILE_EXTENSIONS` make reference to CBU files or a generic file handling?

Reply:
The requirements said that all the files of a taxpayer creation must have a max size of 25MB